Sources/App/routes+documentation.swift

```diff
@@ -169,8 +169,8 @@ extension Request {
                 guard let params = try await DocumentationTarget.query(on: db, owner: owner, repository: repository)?.internal
                 else { throw Abort(.notFound) }
 
-                Current.currentReferenceCache()?[owner: owner, repository: repository] = "\(params.reference)"
-                return .current(referencing: "\(params.reference)")
+                Current.currentReferenceCache()?[owner: owner, repository: repository] = "\(params.docVersion)"
+                return .current(referencing: "\(params.docVersion)")
```

Comment:
We could now also cache `DocVersion`s instead of `String`s but I didn't want to pack that in here.

Reply:
Abandoning this, not worth the trouble.